tracim/tracim/controllers/content.py

```diff
@@ -8,7 +8,11 @@
 
 from cgi import FieldStorage
 from depot.manager import DepotManager
+from preview_generator.exception import PreviewGeneratorException
```

Comment:
as soon as you use has_pdf_preview() method of the preview-generator manager, you can remove the imports (see comment below)